### PR TITLE
Update blt template files to match Drupal 8.2.1.

### PR DESCRIPTION
**Pull request by greylabel**
See: https://github.com/acquia/blt/issues/609

### CODE REVIEW

File: template/docroot/sites/development.services.yml [hunk 1]
```diff
@@ -5,9 +5,3 @@
 services:
   cache.backend.null:
     class: Drupal\Core\Cache\NullBackendFactory
-
```
Comment (grasmash) [inline template/docroot/sites/development.services.yml]: I'd like to keep this settings.

**Comment by grasmash** [comment]
I'd like to commit as few of these as possible. I vaguely remember that template/docroot/sites/default/default.settings.php is required for installation in some instances because of the order that some operations are run. Perhaps that can be changed and we can avoid committing even that.

**Comment by greylabel** [comment]
@grasmash I feel it is very valuable to keep these files in the repo as a reference as they are Core files. Much nicer to be able to refer them locally than needed to google every time.

**Comment by greylabel** [comment]
@grasmash Requested changes made and changed the commit msg to match common syntax being used.

**Comment by grasmash** [comment]
> Much nicer to be able to refer them locally than needed to google every time.

Aren't these files downloaded to your local machine after you run `blt setup`?

**Comment by grasmash** [comment]
I've actually removed all scaffold files: #632 

If you'd like local copies for reference and they're not available at a particular stage of development, please file an issue to that effect. We can look into executing the scaffold file download earlier in the development cycle.

**Pull request closed without merging**